### DISCUSSION TITLE
中文医疗信息助手

### DISCUSSION THREAD

**Discussion by YangZhC** [InternLM/Tutorial#436] [project]
**目的**
本项目将致力于构建一款全面的医疗信息大模型，以解决增强基座模型在医疗领域的表现力。
实现路径：
通过对base模型的微调和利用RAG技术外挂专业知识库，我们的医疗信息大模型将在信息抽取，医疗问答等方法提供高质量、准确且专业的回答。

预期目标：
对患者：对常见医学问题能给出专业回答
对医生：对常见的NLP任务有较好的表现，减轻工作负担（帮助医生总结信息，报告生成，临床术语标准化等）

**Comment by zenith779** [comment]
你好，项目还可以加入吗
